examples/seismic/elastic/elastic_example.py

```diff
@@ -61,16 +61,11 @@ def test_elastic():
                         help="Operator auto-tuning mode")
     args = parser.parse_args()
 
-    # 2D preset parameters
-    if args.dim2:
-        shape = (150, 150)
-        spacing = (10.0, 10.0)
-        tn = 750.0
-    # 3D preset parameters
-    else:
-        shape = (150, 150, 150)
-        spacing = (10.0, 10.0, 10.0)
-        tn = 1250.0
+    # preset parameters
```

Comment:
all comments should start with capital letter

Reply:
Fixed